MiniAn/minian_parameters.py

```diff
@@ -0,0 +1,324 @@
+import os
+import sys
+import inspect
+import numpy as np
+
+# Import for MiniAn lib
+import minian.utilities as mnUtils
+import minian.preprocessing as mnPreproc
+import minian.initialization as mnInit
+import minian.cnmf as mnCnmf
+import minian.motion_correction as mnMotcorr
+
+sys.path.append('..')
+import utilities as utils
+import definitions as defs
+import minian_utilities as mn_utils
+import minian_definitions as mn_defs
+
+class MinianParameters:
+    '''
+    MinianParameters
```

Comment:
Do you leave it for future descriptions?
Also, could you use either `"""` or `'''`

Reply:
Yes I'm leaving it for future descriptions. I added them because my IDE is complaining if class and function don't have a description.... 

Normally I'm using `'''` is it not working?